pep-0458.txt

```diff
@@ -457,6 +457,17 @@ pip for the first time.
 
 __ https://docs.google.com/spreadsheets/d/11_XkeHrf4GdhMYVqpYWsug6JNz5ZK6HvvmDZX0__K2I/edit?usp=sharing
 
+This number of bins SHOULD double when the metadata overhead for returning
+users exceeds 50% (relative to the average size of downloaded packages).
+Presently, this SHOULD happen when the number of targets (simple indices
+and packages) increase at least 4x from over 2M to nearly 9M, at which
```

Comment:
Are there other targets than simple indices or packages? I think it's not necessary to mention it, and this sentence might benefit from being a tad shorter. :)

Reply:
OK